templates/piqa/templates.yaml

```diff
@@ -0,0 +1,121 @@
+dataset: piqa
+templates:
+  1a1d13ee-2ef5-4c80-89dc-b9e003d4ef45: !Template
+    id: 1a1d13ee-2ef5-4c80-89dc-b9e003d4ef45
+    jinja: 'Given a goal and a correct solution, generate a similar but wrong solution.
+
+      Goal: {{goal}}
+
+      Correct solution: {{[sol1, sol2][label]}}
+
+      Wrong solution:
+
+      |||
+
+      {{[sol1, sol2][1 - label]}}
+
+      '
+    name: Generate a similar but wrong solution
+    reference: ''
+  45773b41-4d13-4ade-9102-afe1543f5776: !Template
+    id: 45773b41-4d13-4ade-9102-afe1543f5776
+    jinja: 'Given a goal and 2 solutions, choose the most appropriate solution.
+
+      Goal: {{goal}}
+
+      {{"Solution 1"}}: {{sol2}}
+
+      {{"Solution 2"}}: {{sol1}}
+
+      Answer:
+
+      |||
+
+      {{[sol1, sol2][label]}}'
+    name: 'choose the most appropriate solution: reorder solution'
+    reference: ''
+  5f4b4645-9438-4375-9062-083130e6d04e: !Template
+    id: 5f4b4645-9438-4375-9062-083130e6d04e
+    jinja: "Given a goal and a wrong solution, rewrite it to give a correct solution.\n\
+      Goal: {{goal}} \nSolution: {{[sol1, sol2][1 - label]}}\nCorrected solution:\n\
+      |||\n{{[sol1, sol2][label]}}\n"
+    name: Correct the solution
+    reference: ''
+  99565244-4eaf-4004-a28b-4362ba5bcac3: !Template
+    id: 99565244-4eaf-4004-a28b-4362ba5bcac3
+    jinja: '{{goal}} {{sol2}}
+
+      Does this solution make sense?
+
+      |||
+
+      {{["No", "Yes"][label]}}'
```

Comment:
```
When boiling butter, when it's ready, you can Pour it onto a plate
Does this solution make sense?
```

"Solution" reads strangely to me. Maybe just say "does this phrase make sense?" or alternatively say "Given the problem {{prob}} is {{sol}} as solution to that problem?"